contrib/mesos/pkg/minion/server.go

```diff
@@ -333,6 +336,10 @@ func (ms *MinionServer) AddExecutorFlags(fs *pflag.FlagSet) {
 
 	// hack to forward log verbosity flag to the executor
 	fs.Int32Var(&ms.logVerbosity, "v", ms.logVerbosity, "log level for V logs")
+	
+	// hacks to forward selected cadvisor flags to the executor
+	fs.StringVar(&ms.housekeepingInterval, "housekeeping_interval", ms.housekeepingInterval, "Interval between container housekeepings")
```

Comment:
@jdef Thanks - will rework.   How can I trigger the smoke test myself locally?  I have been trying to run:
hack/build-go.sh
hack/test-go.sh -- is this same as the smoke test?